src/languages.js

```diff
@@ -30,6 +30,10 @@ Languages.get = function(code, key, callback) {
 	var languageData;
 
 	fs.readFile(path.join(__dirname, '../public/language/', code, key), { encoding: 'utf-8' }, function(err, data) {
+		if (err) {
```

Comment:
Yeah the correct thing to do would be to check err.code like `e.code === 'ENOENT'`